.coderabbit.yml

```
@@ -0,0 +1,16 @@
+language: "ko-KR"
+early_access: false 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)